sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -1339,7 +1339,7 @@
       "get": {
         "summary": "Query an IoT hub to retrieve information regarding jobs using the IoT Hub query language",
         "description": "Query IoT hub to retrieve information regarding jobs using the IoT Hub query language. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language for more information.",
-        "operationId": "JobClient_QueryJobs",
+        "operationId": "Jobs_Query",
```

Comment:
We need to clarify with Andrei if this query is for all jobs, just for import/export jobs, or just for scheduled jobs. The name is a little too vague to understand that right now

Reply:
Sure let me ask this to Andrei.

Reply:
Got answer here, it is only scheduled jobs. so perhaps call it Jobs_QueryScheduled? is there better suggestion?

Reply:
Based on above discussions, should we just distinguish v1 and v2 using ExportImportJob and ScheuduledJob as the groupings?

Reply:
added changes